testing/README.md

```diff
@@ -0,0 +1,76 @@
+### E2E tests
+
+#### Dependencies
+
+[oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+### Try it locally
+
+0. Setup vault
+`oblt-cli` stores cluster secrets in vault. 
+Export the environment variable `VAULT_TOKEN` or have the `~/.vault-token` file created.
+For further details about how to configure Vault check [the Vault documentation](https://github.com/elastic/infra/tree/master/docs/vault)
+
+1. Install and configure [oblt-cli](https://studious-disco-k66oojq.pages.github.io)
+
+2. Create a cluster for e2e tests:
+
+In the `testing` directory:
+
+```
+GITHUB_TOKEN=... SLACK_CHANNEL=... make create-cluster
+```
+
+Specify a `GITHUB_TOKEN` with configured SSO and the `SLACK_CHANNEL` - the slack channel identifier to send messages about cluster state. 
```

Comment:
In step 1 above ("Install and configure [oblt-cli](https://studious-disco-k66oojq.pages.github.io)"), we specify a slack channel via `oblt-cli configure --slack-channel`.  Do we need to specify it again in this step via the `SLACK_CHANNEL` environment variable?